I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 208, in script
    "Délinquant 1" "Réponds-moi *paf*" with sshake # tremblement de caméra
  File "game/script.rpy", line 208, in <module>
    "Délinquant 1" "Réponds-moi *paf*" with sshake # tremblement de caméra
NameError: name 'sshake' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 208, in script
    "Délinquant 1" "Réponds-moi *paf*" with sshake # tremblement de caméra
  File "C:\Users\hugof\Desktop\renpy-7.3.5-sdk\renpy\ast.py", line 707, in execute
    say_menu_with(self.with_, renpy.game.interface.set_transition)
  File "C:\Users\hugof\Desktop\renpy-7.3.5-sdk\renpy\ast.py", line 537, in say_menu_with
    what = renpy.python.py_eval(expression)
  File "C:\Users\hugof\Desktop\renpy-7.3.5-sdk\renpy\python.py", line 2059, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\hugof\Desktop\renpy-7.3.5-sdk\renpy\python.py", line 2052, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 208, in <module>
    "Délinquant 1" "Réponds-moi *paf*" with sshake # tremblement de caméra
NameError: name 'sshake' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Espers 1.0
Wed Oct 21 14:49:45 2020
